### ISSUE TITLE
Restart numbering 

### ISSUE THREAD

**Issue by znat** [krilor/heading-numbers#5]
Hi and thanks for this module.
Is there a way to restart numbering?

**Comment by krilor** [comment]
Hey!
I'm not really sure what you mean by restart numbering. Could you elaborate?

Thanks!

**Comment by znat** [comment]
Sorry! I mean that numbering keeps incrementing even after upper level heading has changed..see the example below. 

1. H1
1.1 h1.1
2. H2
2.2 h 2.1

**Comment by krilor** [comment]
If I understand you correctly, what you are after is the following:

1. Chapter
  1.1. Subheading
2. Chapter
  2.2. Subheading

But currently you are getting

1. Chapter
  1.1. Subheading
2. Chapter
  2.1. Subheading

Am I correct?

If that is the case, then that is not possible, I'm afraid.

**Comment by znat** [comment]
No, it's the contrary. Subheadings numbers keep incrementing even after a
new  upper level heading

**Comment by krilor** [comment]
Huh, thats weird. I just tested it. Are you sure that you have the correct structure (using Heading 1, 2 ... etc)?

Could you share a sample document?

**Comment by farside268** [comment]
Sorry to revive this, but I'm having the same problem. What I what to get is:

I. Heading 1
A. Heading 2
II. Heading 1
A. Heading 2

Instead, I get
I. Heading 1
A. Heading 2
II. Heading 1
B. Heading 2

**Comment by farside268** [comment]
I found a cludgy way to fix it for what I need. I commented out "if(prefs.inherit){" at line 326, so now the script always resets regardless of custom settings. You also have to comment out the closing curly bracket.
